vic/vic_run/src/surface_fluxes.c

```diff
@@ -261,6 +261,7 @@ surface_fluxes(bool                 overstory,
     coverage = snow->coverage;
     snow_energy = (*energy);
     soil_energy = (*energy);
+    iter_soil_energy = (*energy);
```

Comment:
@jhamman : You asked me to review this, but since I don't know what it is here for I have not idea whether it is right or not (no context for review). Is this part of an intel warning (as the PR says)?